### ISSUE TITLE
Post-FxA-Upgrade: hookup debug item for custom servers

### ISSUE THREAD

**Issue by garvankeeley** [mozilla-mobile/firefox-ios#6421]
I see there is a constructor FxAConfig in the a-s side that is ready for use
```
open class FxAConfig {
public init(contentUrl: String, clientId: String, redirectUri: String, tokenServerUrlOverride: String? = nil)
```

**Comment by vladikoff** [comment]
@garvankeeley yea no rush, I was just trying to research the issue in https://github.com/mozilla-mobile/firefox-ios/issues/6407

**Comment by csuciu** [comment]
Verified as fixed on master 7a75a0c3.
